### ISSUE TITLE
List API doesn't work for new generated project 

### ISSUE THREAD

**Issue by yatendragoel** [jhipster/generator-jhipster#8296]
##### **Overview of the issue**

List API `/api/users?page=1&size=20` doesn't work.

If you create any new entity using the entity sub-generator, its List API also doesn't work

##### **JHipster configuration**

$ jhipster info
Using JHipster version installed globally
Executing jhipster:info
Options:
Welcome to the JHipster Information Sub-Generator

##### **JHipster Version(s)**

```
calorie-counter-backend@0.0.0 E:\workspace\sts_workspace\CalorieCounterBackend
`-- (empty)

```

##### **JHipster configuration, a `.yo-rc.json` file generated in the root folder**

<details>
<summary>.yo-rc.json file</summary>
<pre>
{
  "generator-jhipster": {
    "promptValues": {
      "packageName": "com.example",
      "nativeLanguage": "en"
    },
    "jhipsterVersion": "4.14.3",
    "baseName": "CalorieCounterBackend",
    "packageName": "com.example",
    "packageFolder": "com/example",
    "serverPort": "8080",
    "authenticationType": "jwt",
    "cacheProvider": "no",
    "enableHibernateCache": false,
    "websocket": false,
    "databaseType": "mongodb",
    "devDatabaseType": "mongodb",
    "prodDatabaseType": "mongodb",
    "searchEngine": false,
    "messageBroker": false,
    "serviceDiscoveryType": false,
    "buildTool": "maven",
    "enableSocialSignIn": false,
    "enableSwaggerCodegen": false,
    "jwtSecretKey": "replaced-by-jhipster-info",
    "enableTranslation": true,
    "applicationType": "monolith",
    "testFrameworks": [],
    "jhiPrefix": "jhi",
    "nativeLanguage": "en",
    "languages": [
      "en",
      "de"
    ],
    "clientPackageManager": "yarn",
    "skipClient": true
  }
}
</pre>
</details>

##### **JDL for the Entity configuration(s) `entityName.json` files generated in the `.jhipster` directory**

<details>
<summary>JDL entity definitions</summary>

<pre>

</pre>
</details>

##### **Environment and Tools**

java version "1.8.0_172"
Java(TM) SE Runtime Environment (build 1.8.0_172-b11)
Java HotSpot(TM) Client VM (build 25.172-b11, mixed mode, sharing)

git version 2.12.2.windows.2

node: v8.11.4

npm: 6.4.0

yarn: 1.3.2

Docker version 18.03.0-ce, build 0520e24302

docker-compose version 1.20.1, build 5d8c71b2

Congratulations, JHipster execution is complete!

**Comment by gmarziou** [comment]
> "jhipsterVersion": "4.14.3"

Any reason, you don't use Jhipster 5?

**Comment by ruddell** [comment]
I can't reproduce.  If you are unauthenticated then it gives a 401 as expected.  If you are authenticated, it returns the list of users.

You are also querying for page 1 which won't have any results, there are only 4 users at start so you need to query for page 0

**Comment by yatendragoel** [comment]
Yes, there is something strange here. It works sometimes and it doesn't work sometimes. This even surprised me as well because an error in the list API is something which would have been caught by now. Also, all the code looks perfectly fine to me.

I will figure out the root of cause to it one I get more time. For now, let's keep it closed.

**Comment by jdubois** [comment]
Thanks @yatendragoel - don't hesitate to comment or reopen!
